README.md

```diff
@@ -32,25 +32,22 @@ The behavior of the application is similar to that of [Thebe](https://github.com
 The following libraries are known to work with the dashboard server:
 
 * jupyter_dashboards 0.5.x
-* jupyter_dashboards_bundlers 0.7.x 
+* jupyter_dashboards_bundlers 0.7.x
 * ipywidgets 5.x
 * jupyter_declarativewidgets 0.5.x
 * Bokeh 0.11.x
 * Plotly 1.9.x
 
-## Try It
+## Install it
 
-If you want to see the dashboard server in action, you can run a handful of demos we included in this project in a Docker container. After setting up Docker (e.g. using [docker-machine](https://docs.docker.com/machine/get-started/)), do the following in a git clone of this repo:
+You can install the dashboard server using `npm`. See the next section about how to install and configure the other prerequisite components.
 
 ```
-make build
-make examples
-make demo-container
+npm install -g jupyter-dashboards-server
+jupyter-dashboards-server --help
```

Comment:
`--help` just spits out the config file, but doesn't tell user how to specify those settings when starting on the command line.